dist/Tie-File/t/44_binmode.t

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use 5.008;
+
+# option to specify binmode
+
+use Tie::File;
+use Test::More tests => 3;
+use Test::More;
+
+use utf8;
+use feature 'unicode_strings';
```

Comment:
This feature doesn't appear to be needed in this test. For now I think it could be removed.